modules/ROOT/pages/advanced-templates.adoc

```diff
@@ -0,0 +1,103 @@
+= Advanced Template plugin
+:navtitle: Advanced Template
+:description: Create and edit complex templates in {productname} for use with {productname}.
+:description_short: Create templates.
+:keywords: plugin, advanced template, templates, advtemplate.
+:pluginname: Advanced Template
+:plugincode: advtemplate
+:plugincategory: premium
+
+include::partial$misc/admon-premium-plugin.adoc[]
+
+include::partial$misc/admon-requires-6.4v.adoc[]
+
```

Comment:
```suggestion
include::partial$misc/admon-requires-6.4v.adoc[]

NOTE: The documentation for this page is a work in progress. Further details on the {pluginname} plugin will be added to this page soon.
```
Might be important to forewarn that this page is a WIP, and to check back later for more information